packages/mobile/src/tests/joining.process.test.tsx

```diff
@@ -54,14 +54,14 @@ describe('Joining process', () => {
     const processText = screen.getByTestId('connection-process-text')
     expect(processText.props.children).toEqual('Connecting process started')
 
-    store.dispatch(connection.actions.setTorConnectionProcess(ConnectionProcessInfo.INITIALIZING_LIBP2P))
+    store.dispatch(connection.actions.setConnectionProcess(ConnectionProcessInfo.INITIALIZING_IPFS))
     await act(async () => {})
 
     const processText2 = screen.getByTestId('connection-process-text')
     console.log(processText2.props)
-    expect(processText2.props.children).toEqual('Initializing libp2p')
+    expect(processText2.props.children).toEqual('Initialized backend modules')
```

Comment:
Generally in tests I'd use variable (like BACKEND_MODULES in this case) instead of hardcoding string.